client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -333,6 +343,18 @@ class CampaignAdminList extends Component {
       );
     }
 
+    if (error) {
+      return (
+        <div className={previewClasses}>
+          {
+            errorMessage === 'Forbidden'
```

Comment:
Presuming we can get the status code of the request here, I would use that to decide what message to show. I would also include a third "Something went wrong" catch-all message something completely different went wrong.